src/commands/code/push.ts

```diff
@@ -15,6 +15,7 @@ const MESSAGES = {
   appId: APP_ID_TO_ENTER,
   force: 'Force push to live version',
   'client-side': 'Push files to CDN',
+  'security-scan': 'Run security scan during deployment',
```

Comment:
Lets be more verbose and user-facing here. is it only scanning dependencies? I dont rememebr, but if yes:

```suggestion
  'security-scan': 'Run a security scan to find dependency vulnerabilities during code deployment',
```